.travis.yml

```diff
@@ -1,50 +1,100 @@
+# Variables needed for this script are:
+
+# Case 1: Stored on Travis Settings
+  # For all environments:
+    # - DOCKER_CI_REPO_NAME
+    # - DOCKER_USERNAME
+    # - DOCKER_PASSWORD
+
+  #For dev environment:
+    # - AWS_ACCOUNT_USER_ID_DEV
+    # - AWS_ACCOUNT_SECRET_DEV
+    # - AWS_PROFILE_DEV <-- is this needed?
+
+  #For Prod environment:
+    # - AWS_ACCOUNT_USER_ID_PROD
+    # - AWS_ACCOUNT_SECRET_PROD
+    # - AWS_PROFILE_PROD <-- is this needed?
+
+  #For Demo environment:
+    # - AWS_ACCOUNT_USER_ID_DEMO
+    # - AWS_ACCOUNT_SECRET_DEMO
+    # - AWS_PROFILE_DEMO <-- is this needed?
+
+# Case 2: Created when Travis CI runs
+    # - IMAGE_TAG
+
 sudo: required
```

Comment:
`sudo: required` is deprecated in Travis CI: we can safely remove this line:
See [Travis View confif comment for more details](https://travis-ci.org/Unee-T-INS/frontend/jobs/636699944/config#rccb_frontend:.travis.yml@2bca01be674570b8.27)

Reply:
fix in commit: https://github.com/Unee-T-INS/frontend/pull/7/commits/bc2335722d03b53820aee8c35b28f2aff72aab77